derive/tests/pubsub-macros.rs

```diff
@@ -22,17 +22,21 @@ type Result<T> = ::std::result::Result<T, MyError>;
 pub trait Rpc {
 	type Metadata;
 
-	/// Hello subscription
+	/// Hello subscription.
 	#[pubsub(subscription = "hello", subscribe, name = "hello_subscribe", alias("hello_alias"))]
 	fn subscribe(&self, a: Self::Metadata, b: Subscriber<String>, c: u32, d: Option<u64>);
 
 	/// Unsubscribe from hello subscription.
 	#[pubsub(subscription = "hello", unsubscribe, name = "hello_unsubscribe")]
 	fn unsubscribe(&self, a: Option<Self::Metadata>, b: SubscriptionId) -> Result<bool>;
 
-	/// A regular rpc method alongside pubsub
+	/// A regular rpc method alongside pubsub.
 	#[rpc(name = "add")]
 	fn add(&self, a: u64, b: u64) -> Result<u64>;
+
+	/// A notification alongside pubsub.
+	#[rpc(name = "notify")]
```

Comment:
Can we get a test for notifications to make sure that it actually only accepts a request with no id for that method?

Reply:
Sure! I'll add a test for that.

Reply:
Okay, I've added a new test case `should_accept_only_notifications()` which should cover this.